frontend/angular/README.md

```diff
@@ -1,14 +1,23 @@
 # Angular Tech Guides [Tech Guides under construction]
 This section is still under construction. 
 
-## Recommended Libraries
+## Content
 
-- [Internal Libraries]
+- [Project Bootstrapper](#project-bootstrapper)
+- [Style Guide](#style-guide)
+- [Internal Libraries](#internal-libraries)
+- [External Libraries](#external-libraries)
+- [Useful Documentation](#useful-documentation)
+- [Posts](#posts)
```

Comment:
delete this, all angular posts should be in the front-end readme with the prefix `[Angular]`